op-node/p2p/gossip.go

```diff
@@ -362,13 +362,13 @@ func BuildBlocksValidator(log log.Logger, cfg *rollup.Config, runCfg GossipRunti
 
 		if blockVersion.HasBlobProperties() {
 			// [REJECT] if the block is on a topic >= V3 and has a blob gas used value that is not zero
-			if payload.BlobGasUsed == nil || (!cfg.IsL2BlobEnabled() && *payload.BlobGasUsed != 0) {
+			if payload.BlobGasUsed == nil || (!cfg.IsL2Blob(uint64(payload.Timestamp)-cfg.BlockTime) && *payload.BlobGasUsed != 0) {
```

Comment:
Why we need to subtract the blockTime (2s) here?

Reply:
Because the geth side is always checking against the parent block to test activation of the current block. That's why the parameter for `IsL2Blob` is named `parentTime`.